content/en/docs/usage/kraft_advanced.md

```diff
@@ -0,0 +1,609 @@
+
+```text
+Usage: kraft init [OPTIONS] [NAME]
+
+  Initializes a new unikraft application.
+
+  Start here if this is your first time using (uni)kraft.
+
+Options:
+  -t, --template NAME  Use an existing application as a template.
+  -p, --plat PLAT      Target platform.
+  -m, --arch ARCH      Target architecture.
+  -w, --workdir PATH   Specify an alternative directory for the application
+		       [default is cwd].
+  -M, --with-makefile  Create a Unikraft compatible Makefile.
+  -D, --no-deps        Do not download additional dependencies for application
+		       components.
+  -d, --dump           Dump dependencies into project directory.
+  -F, --force          Overwrite any existing files.
+  -h, --help           Show this message and exit.
+```
+
+## `kraft configure`
+
+You can configure the application by using `kraft configure`.
+You can specify the platform and architecture by using the `-p, --plat` and `-m --arch` flags.
+You can enable, disable and set a value for an option by using `-y, --yes`, `-n, --no`, `-s, --set`.
+You can change the directory for your application by using the `-w, --workdir` flags.
+You can force the overwriting of the existing configuration by using `-F, --force`.
+
+```
+$ kraft configure -p kvm -m x86_64 -s CONFIG_LIBUKMMAP=y -w /home/stefan/projects/unikraft/hello/ -F
+make: Entering directory '/home/stefan/.unikraft/unikraft'
+[...]
+#
+# configuration written to /home/stefan/projects/unikraft/hello/.config
+#
+make: Leaving directory '/home/stefan/.unikraft/unikraft'
+```
+
+### Overview of `kraft configure` command
+
+```text
+Usage: kraft configure [OPTIONS]
+
+  Configure the unikernel using the KConfig options set in the kraft.yaml
+  file.  Alternatively, you can use the -k|--menuconfig flag to open the TUI
+  to manually select the configuration for this unikernel.
+
+  When the unikernel is configured, a .config file is written to the working
+  directory with the selected KConfig options.
+
+Options:
+  -p, --plat PLAT         Target platform.
+  -t, --target TARGET     Target name.
+  -m, --arch ARCH         Target architecture.
+  -F, --force             Force writing new configuration.
+  -k, --menuconfig        Use Unikraft's ncurses Kconfig editor.
+  -w, --workdir PATH      Specify an alternative directory for the application
+			  [default is cwd].
+  -y, --yes KOPTION       Specify an option to enable.
+  -n, --no KOPTION        Specify an option to disable.
+  -s, --set KOPTION       Set an option's value.
+  -u, --use-version COMP  Use the specified version for the component, e.g.
+			  -u unikraft@staging (will override kraft.yaml).
+  -h, --help              Show this message and exit.
+```
+
+## `kraft build`
+
+You can use `kraft build` to build your application.
+You can use the `--fetch / --no-fetch` and `--prepare / --no-prepare` flags to run the fetch and prepare steps before building.
+You can use `-j, --fast` to use all CPU cores for building the app.
```

Comment:
Are you sure about this? Wouldn't it just pass `-j` to `make`? And that's a bit different:

```
If the -j option is given without  an argument, make will not limit the number of jobs that can run simultaneously.
```

This is why I use `-j $(nproc)`.

Reply:
You're right, it will [just pass the `-j` flag](https://github.com/unikraft/kraft/blob/staging/kraft/app/app.py#L223), updated.